### ISSUE TITLE
Batch Normalization | LOUIS' BLOG

### ISSUE THREAD

**Issue by isLouisHsu** [isLouisHsu/isLouisHsu.github.io#22]
https://louishsu.xyz/2019/08/27/Batch-Normalization/ 

 前言深度学习网络可通过mini-batch随机梯度下降的方式进行优化，然而在优化过程中，存在梯度消失、收敛不稳定的问题。 原理机器学习中，假定训练数据与测试数据独立同分布(independent and identical independent)，当输入数据进行白化操作后，可使模型收敛速度加快，这被称作Covariate Shift。 而在神经网络内部，各层的特征图数据分布不均匀，称为Inter
